Automattic/wp-calypso
# Signup reskin: Integration PR

#### Changes proposed in this Pull Request

This PR replaces the legacy experiments platform with the new ExPlat JS client for the reskinned signup experiment. Please note that the UI changes for this experiment are implemented in subsequent PRs. This PR only swaps out the legacy experiment platform for the new one.

For more context on the experiment, read pbxNRc-IO-p2. 

#### Testing instructions



* Assign yourself to treatment from Abacus, and visit /start
* Verify that the reskinned signup UI loads(white background)

<img width="1677" alt="Screenshot 2021-03-31 at 12 11 13 AM" src="https://user-images.githubusercontent.com/1269602/113050203-3394ba00-91c2-11eb-8ad5-dec3e5a92203.png">

* Assigning yourself to control should load the regular blue background UI in /start.
